docs/source/pages/howToWorkWithPlatform/adminApplication/integration/integrationPage.rst

```diff
@@ -45,7 +47,7 @@ Customer can select only one of three default rules. When customer select 3 rule
 Conversion rules list(5)
 ========================
 
-By default conversion rule list is empty. For create new rule need click on "Add new rule" button, after that we redirect customer to create conversion rule.
+By default conversion rule list is empty. To create new rule just click on "Add new rule" button, after that customer will be redirected to conversion rule creation form.
```

Comment:
better to say 'you will be redirected' because sentence starts like this (just click on button, not customer should click on button)